test/scriptsSpec.js

```diff
@@ -1,2459 +0,0 @@
-/* eslint-disable
```

Comment:
You've split and modified all code in one patch. This makes code review really hard.

Perhaps you can split this into two patches, or even two PRs.